Monika After Story/game/script-topics.rpy

```diff
@@ -4695,3 +4695,69 @@ label monika_100k:
     else:
         m 1m "Almost~"
     return
+
+
+Init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_beauty",category=['Misc'],prompt="Beauty Standards",random=True))
+    label monika_beauty:
```

Comment:
labels should have no indents. Remove one level of indentation from labels (and the dialogue in them).